### PR TITLE
Add KLayout DRC step

### PR DESCRIPTION
**Pull request by kareefardi**
* Updated Open PDKs to `cd1748b`
* All steps:
  * Rename `PRIMARY_SIGNOFF_TOOL` to `PRIMARY_GDSII_STREAMOUT_TOOL`
* Integrated KLayout DRC for sky130 (skips for other PDKs)
* Created new step `KLayout.DRC`
  * Compatible with sky130 (skips other PDKs)
  * Adds two new PDK variables, `KLAYOUT_DRC_RUNSET` and `KLAYOUT_DRC_OPTIONS`
---

https://github.com/efabless/openlane2-step-unit-tests/pull/15

### CODE REVIEW

File: openlane/steps/klayout.py [hunk 1]
```diff
@@ -346,7 +346,7 @@ class DRC(KLayoutStep):
         ),
         Variable(
             "KLAYOUT_DRC_OPTIONS",
-            Optional[Dict[str, Union[bool, int]]],
+            [Dict[str, Union[bool, int]]],
```
Comment (donn) [inline openlane/steps/klayout.py:349]: shouldn't be an array (for when you wake up)

Comment (kareefardi) [reply openlane/steps/klayout.py:349]: Why array?

Comment (donn) [reply openlane/steps/klayout.py:349]: Enclosed by `[]`

**Review by donn — APPROVED** [review]

**Comment by donn** [comment]
Gonna review it a bit more in-depth tomorrow then merge, hope that's fine

**Pull request merged**